src/components/MyAccount/Settings/AccountSettingsButtons.tsx

```diff
@@ -0,0 +1,55 @@
+import { Icon, Button } from "@nypl/design-system-react-components"
+import type { Dispatch } from "react"
+
+interface AccountSettingsButtonsPropsType {
+  currentlyEditing: boolean
+  setCurrentlyEditing: Dispatch<React.SetStateAction<boolean>>
+}
+
+const AccountSettingsButtons = ({
+  currentlyEditing,
+  setCurrentlyEditing,
+}: AccountSettingsButtonsPropsType) => {
+  const toggleCurrentlyEditing = () => setCurrentlyEditing(!currentlyEditing)
```

Comment:
Use the value from the callback argument to update the value: `setCurrentlyEditing((prevValue) => !prevValue)`